### PR TITLE
feat: cancel order and prescription

### PR DESCRIPTION
**Pull request by jknapp25**
- cancel order
- cancel prescription
- align order details and fulfillment table columns
- CopyText comp

### CODE REVIEW

File: apps/app/src/views/routes/Prescription.tsx [hunk 1]
```diff
@@ -70,21 +72,49 @@ export const Prescription = () => {
...
-          <Button
-            aria-label="New Order"
-            as={RouterLink}
-            to={`/orders/new?patientId=${patient.id}&prescriptionId=${id}`}
-          >
-            Create Order
-          </Button>
-        )}
+        <HStack>
+          {loading ? (
```
Comment (samkotlove) [inline apps/app/src/views/routes/Prescription.tsx:81]: These buttons (included create order), should only show if prescription is ACTIVE

Comment (jknapp25) [reply apps/app/src/views/routes/Prescription.tsx:115]: added disabling to both

File: apps/app/src/views/routes/Order.tsx [hunk 2]
```diff
@@ -85,11 +121,73 @@ export const Order = () => {
...
+        {loading ? (
+          <Skeleton width="112px" height="32px" borderRadius="md" />
+        ) : (
+          <Button
+            size="sm"
+            aria-label="Cancel Order"
+            isDisabled={order.fulfillment?.type !== types.FulfillmentType.MailOrder}
+            onClick={async () => {
+              const decision = await confirmWrapper('Cancel this order?', {
+                description: 'You will not be able to recover this order.',
```
Comment (samkotlove) [inline apps/app/src/views/routes/Order.tsx:168]: Feel like we can modify the copy a little, maybe "You will not be able to undo this" or something?

Comment (jknapp25) [reply apps/app/src/views/routes/Order.tsx]: updated

**Review by samkotlove — APPROVED** [review]

**Pull request merged**